### ISSUE TITLE
Small error?

### ISSUE THREAD

**Issue by YacratesWyh** [llSourcell/AI_For_Music_Composition#3]
main.py line 26                os.path.realpath(__file__),os.path.pardir, 'training_data',
there could need a pardir in the middle.

**Comment by ynnsnr** [comment]
Yes, removing the ```os.path.realpath(__file__)``` argument worked for me
